apps/frontend/src/app/5_pages/ConvertPage/ConvertPage.tsx

```diff
@@ -256,24 +279,81 @@ const ConvertPage: FC = () => {
 
             <div className="w-full flex flex-row justify-between items-center gap-3 mt-3.5">
               <AmountInput
-                value={amount}
+                value={renderDestinationAmount}
                 label={t(commonTranslations.amount)}
                 readOnly
-                placeholder="0"
+                placeholder={t(commonTranslations.na)}
                 className="w-full flex-grow-0 flex-shrink"
                 dataAttribute="convert-to-amount"
               />
               <Select
                 value={destinationToken}
                 onChange={setDestinationToken}
                 options={destinationTokenOptions}
-                labelRenderer={() => getAssetRenderer(destinationToken)}
                 className="min-w-[6.7rem]"
                 dataAttribute="convert-to-asset"
               />
             </div>
           </div>
 
+          <Accordion
+            className="mt-4 mb-3 text-xs"
+            label={t(translations.common.advancedSettings)}
+            open={advanced}
+            onClick={() => setAdvanced(!advanced)}
+            dataAttribute="convert-settings"
+          >
+            <div className="mt-2 mb-4">
+              <AmountInput
+                value={defaultSlippageTolerance}
+                label={t(translations.convertPage.slippageTolerance)}
+                className="max-w-none w-full"
+                unit="%"
+                placeholder="0"
+                readOnly
+              />
+            </div>
+          </Accordion>
+
+          <SimpleTable className="mt-3">
+            <SimpleTableRow
+              label={t(translations.convertPage.price)}
+              valueClassName="text-primary-10"
+              value={
+                <AmountRenderer
+                  value={1}
+                  suffix={sourceToken}
+                  precision={TOKEN_RENDER_PRECISION}
+                />
+              }
+            />
+          </SimpleTable>
+
+          <SimpleTable className="mt-3">
```

Comment:
This section is visible only if the advanced settings section should be visible.